Week3/homework/step2-6.js

```diff
@@ -4,13 +4,24 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const result = arr.reduce((total, item) => total.concat(item));
+  return result;
 }
 
-function flattenArray3d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+function flattenArray3d(arr, result = []) {
+  // const result = arr;
+  // for (let i = 0; i < 2; i++) {
+  //   result.reduce((accu, curr) => accu.concat(curr));
+  // }
```

Comment:
You should remove commented-out code from a pull request for the same reason that you should not crossed-out text in your CV. A pull request is a request to a reviewer to approve your work. You shouldn't bother him/her with things you tried but rejected.